views/admin/requests/index.js

```diff
@@ -319,7 +319,8 @@ exports.read = function(req, res, next) {
     var outcome = {};
 
     var getRecord = function(callback) {
-        req.app.db.models.Request.findById(req.params.id).exec(function(err, record) {
```

Comment:
tl;dr: let's revert the change to this line.

It looks like this function would be called when a user views an individual smoke alarm request, but unfortunately we're not allowing them to do that yet.  I think `getRecord()` is only in here because it came in as part of the Drywall skeleton (long story @raymondberg).  Moreover, I'm pretty sure this code was actually broken before this change (we expect `req.app.db.Request` instead of `req.app.db.models.Request`, among other things), and I'd rather not have additional changes on top of broken code.

Finally, I think the intention here was for this function to take the db id, not the serial/public id, so in order to change that we'd probably want to do something like:

```
req.app.db.Request.findOne( 
     {where: {public_id: req.params.id} 
      }).then( function (request) { 
        // do something
     });
```